smdebug/tensorflow/context_manager.py

```diff
@@ -0,0 +1,10 @@
+class ProfilerContextManager(object):
+    def __init__(self, hook, mode):
+        self.hook = hook
+        self.mode = mode
+
+    def __enter__(self):
+        self.hook.profiling_start_batch(self.mode)
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.hook.profiling_end_batch(self.mode)
```

Comment:
Check if we can eliminate the need for a separate class for a context manager?

Reply:
Will remove the class and just use `contextlib.context_manager`